scene/gui/code_edit.cpp

```diff
@@ -2412,6 +2412,68 @@ Ref<Texture2D> CodeEdit::_get_folded_eol_icon() const {
 	return theme_cache.folded_eol_icon;
 }
 
+/* Text manipulation */
+void CodeEdit::duplicate_lines() {
```

Comment:
Seems to be from #81551, but could you move this above `/* Visual */` so it's the same order as the header?

Otherwise looks great, really appreciate the unit tests too!

Reply:
Done